### PR TITLE
Remove commit load branches and tags in wiki repo

### PR DESCRIPTION
**Pull request by yp05327**
If click `load branches and tags`, you will get 500 error from backend, as it is a wiki repo.
image
image

### CODE REVIEW

File: templates/repo/commit_load_branches_and_tags.tmpl [hunk 1]
```diff
@@ -1,3 +1,4 @@
+{{if not .PageIsWiki}}
```
Comment (delvh) [inline templates/repo/commit_load_branches_and_tags.tmpl:1]: Yes, that's one approach.
Another would be to show the information for the wiki repo when present.
The only thing we would need to change in that case is line 4.

**Review by delvh — APPROVED** [review]

File: templates/repo/commit_load_branches_and_tags.tmpl (hunk 1 above)
Comment (delvh) [reply templates/repo/commit_load_branches_and_tags.tmpl:1]: (and add an additional route for the wiki repo)

Comment (yp05327) [reply templates/repo/commit_load_branches_and_tags.tmpl:1]: Which is better?
IMO, wiki's branches and tags are generated by Gitea, and should not show them to the end user.

**Review by lunny — APPROVED** [review]

File: templates/repo/commit_load_branches_and_tags.tmpl (hunk 1 above)
Comment (yp05327) [reply templates/repo/commit_load_branches_and_tags.tmpl:1]: emmm, it seems that we can clone it, and push changes to it. So maybe display the correct info is better.
image

**Pull request merged**